components/automate-gateway/gateway/services.go

```diff
@@ -683,6 +683,58 @@ func (s *Server) ReportExportHandler(w http.ResponseWriter, r *http.Request) {
 	writeContent(w, stream, query.Type)
 }
 
+func(s *Server) NodeFetchHandler(w http.ResponseWriter, r *http.Request){
```

Comment:
This is not the way that we create an API unless we have a need. We should follow defining the proto in external API and exposing the end points using GRPC gateway. @Dmaddu @mchengat88